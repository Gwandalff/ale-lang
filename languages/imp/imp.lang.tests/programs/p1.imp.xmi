<?xml version="1.0" encoding="UTF-8"?>
<imp:Assign
    xmi:version="2.0"
    xmlns:xmi="http://www.omg.org/XMI"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:imp="http://www.example.org/imp"
    xsi:schemaLocation="http://www.example.org/imp ../../imp.model/model/imp.ecore"
    name="a">
  <exp xsi:type="imp:IntConst"
      value="1"/>
</imp:Assign>
