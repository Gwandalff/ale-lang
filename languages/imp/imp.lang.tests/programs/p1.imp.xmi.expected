<?xml version="1.0" encoding="ASCII"?>
<imp:Store xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:imp="http://imp.imp.imp/">
  <values key="a">
    <value xsi:type="imp:IntValue" value="1"/>
  </values>
</imp:Store>
